app/uk/gov/hmrc/tai/connectors/TaxAccountConnector.scala

```diff
@@ -85,8 +91,8 @@ class TaxAccountConnector @Inject()(
       val requestHeader = headersForUpdate(hc, version, sessionOrUUID, npsConfig.originatorId)
 
       httpHandler
-        .postToApi[List[IabdUpdateAmount]](url, amountList, APITypes.NpsIabdUpdateEstPayManualAPI)(
-          requestHeader,
+        .postToApi[List[IabdUpdateAmount]](url, amountList, APITypes.NpsIabdUpdateEstPayManualAPI, requestHeader)(
+          hc,
```

Comment:
can remove implicits here

Reply:
I need to state `IabdUpdateAmountFormats.formatList`, so I left that in and changed hc to implicitly